css/class-naming-conventions/readme.md

```diff
@@ -68,14 +69,13 @@ The highest level of a module — it should describe an independent module that
 
 ### Sub-components
 
-This is a secondary element inside of a component. It is always written as a chain of its parent component to avoid any inheritance issues. Your subcomponents should be named in a way that keeps them from having to have subcomponents of their own. If you find you need to write a subcomponent for a subcomponent, consider breaking the parent out into its own component.
+This is a secondary element that is child to its parent component. The classname should be formatted as such: `c-[parent-component-name]__[sub-component-name]`. Sub-components do not, and should not, have sub-components of their own. If you find you need to write a sub-sub-component, instead just treat it as a sub-component – sub-components are only ever child to the parent component.
```

Comment:
"A sub-component is a child of its parent component, with a classname like `c-[parent-component-name]__[sub-component-name]."

Should we also mention that if you need to write a bunch of sub-sub-components, you might be making your component too "high-level"?